src/chain.ts

```diff
@@ -152,7 +152,7 @@ export class Chain {
             .on("zoom", () => {
                 subject.next(d3.event.transform);
             });
-        svg.call(zoom);
+        svg.call(zoom).on("dblclick.zoom", null);
```

Comment:
Double clicking anywhere on the container of the chain would zoom in the view a little. I had to disable it otherwise double clicking on links would not work correctly